src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -158,6 +158,11 @@ object ScalafixPlugin extends AutoPlugin {
         SettingKey[Boolean]("bspEnabled") := false
       )
     ),
+    scalafixInterfaceProvider := ScalafixInterface.fromToolClasspath(
+      (scalafixScalaBinaryVersion).value,
+      scalafixDependencies = (scalafixDependencies).value,
+      scalafixCustomResolvers = (scalafixResolvers).value
```

Comment:
nit: unnecessary parens
```suggestion
      scalafixScalaBinaryVersion.value,
      scalafixDependencies = scalafixDependencies.value,
      scalafixCustomResolvers = scalafixResolvers.value
```